packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -462,6 +462,10 @@ const helperBehavior = {
       debt,
     );
     const wantRUN = proposal.want.RUN || helper.emptyDebt();
+    if ([giveColl, giveRUN, wantColl].every(a => AmountMath.isEmpty(a))) {
```

Comment:
is this check not needed in runstake too?
https://github.com/Agoric/agoric-sdk/blob/6b54107e38a77a63073d9d422b186ea5ab0ee4e1/packages/run-protocol/src/runStake/runStakeKit.js#L309-L310

Reply:
I added tests to try to replicate the same situation in runStake, and wasn't able to create a problematic situation.  RunStake creates attestations for each trade, and fails when empty attestations are requested, so I don't think the same situation can arise.

Reply:
Re-reading the PR description I see you had noted that. I regret missing it earlier.